pkg/pool-stable/contracts/ComposableStablePoolProtocolFees.sol

```diff
@@ -155,6 +155,14 @@ abstract contract ComposableStablePoolProtocolFees is
         // Each invariant should be larger than its precedessor. In case any rounding error results in them being
         // smaller, we adjust the subtraction to equal 0.
 
+        // Note: in the unexpected scenario where the rates of the tokens shrink over time instead of grow (i.e. if the
+        // yield is negative), the non-exempt growth invariant might actually be *smaller* than the swap fee growth
```

Comment:
You only address non-exempt growth invariants here. For decreases of exempt rate providers, it's possible for the totalGrowthInvariant to be smaller than the totalNonExemptGrowthInvariant.

Reply:
...,the non-exempt growth invariant might actually be *smaller* than the swap fee growth invariant and the total growth invariant might actually be *smaller* than the non-exempt growth invariant.